File: lista3/zadanie2/Kwadrat.cpp
#include "Figura.hpp"
#include "Czworokat.hpp"
#include "Kwadrat.hpp"


double Kwadrat::obliczPole(){
    return dajBok1() * dajBok1();
}

void Kwadrat::dajNazwe(){
    std::cout << "Jestem kwadratem" << std::endl;
}

File: lista3/zadanie1/Kwadrat.java
class Kwadrat extends Czworokat  {

    public Kwadrat(int bok1, int bok2, int bok3, int bok4, int kat){
        super(bok1, bok2, bok3, bok4, kat);
        
    }

    public double obliczPole() {
        return dajBok1() * dajBok4();
    }

    public void dajNazwe(){
        System.out.println("Jestem kwadratem");
    }
    
}
